### ISSUE TITLE
Accessibility Navigation is Broken

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#11393]
### Bug description

The User can't use the keyboard to navigate AU4 UI. 

### Steps to reproduce

1. Launch AU4
2. Try to navigate UI via keytboard 
3. Observe

### Expected behavior

Keyboard navigation should work 

### Actual behavior

Keyboard navigation doesn't work 

### Audacity Version

Audacity 4.0 beta 2

### Operating system

OS independent

**Comment by ianalarina** [comment]
try to bisect once you have a moment, would be good to know when this got broken

**Comment by igorkorsukov** [comment]
It looks like I broke it, I'll fix it now
